ISISComputingGroup/cs-studio
# Ticket 970: Scan Server Transmits Available Commands

To test:
* Run the scan server product
* In a browser go to http://localhost:4810/commands
* Confirm all relevant information about available commands is given in sensible xml
